templates/config.edn

```diff
@@ -102,7 +102,7 @@
  ;; :custom-js-url "https://cdn.logseq.com/custom.js"
 
  ;; Set a custom Arweave gateway
- ;; Default value: https://arweave.net
+ ;; Default gateway: https://arweave.net
```

Comment:
**non-blocking**: `value` is acceptable.

Reply:
Gateway is more descriptive and it was approved in #9287. It was changed in #9408, and I do not think that change was necessary therefore I am reverting it.